api/policy.go

```diff
@@ -32,6 +37,13 @@ const (
 	// PipelineApprovalValueManualAlways means the pipeline should be manually approved by user to proceed.
 	PipelineApprovalValueManualAlways PipelineApprovalValue = "MANUAL_APPROVAL_ALWAYS"
 
+	// AssigneeGroupValueWorkspaceOwnerOrDBA means the assignee is selected from the workspace owners or DBAs.
+	AssigneeGroupValueWorkspaceOwnerOrDBA AssigneeGroupValue = "WORKSPACE_OWNER_OR_DBA"
+	// AssigneeGroupValueProjectOwner means the assignee can be selected from the project owners if the issue type is one of
+	//  - alter schema
+	//  - update data
+	AssigneeGroupValueProjectOwner AssigneeGroupValue = "PROJECT_OWNER"
```

Comment:
Instead of using an enum, shall we use a struct because different issue types will have different corresponding values?

Reply:
Done, PTAL.